modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridLocalConfigManager.java

```diff
@@ -298,6 +315,62 @@ private void addCacheFromConfiguration(
         }
     }
 
+    /**
+     * Validates already processed cache configuration instead newly defined.
+     *
+     * @param cacheConfigs Already processed caches.
+     * @param cfg Currently processed cache config.
+     * @return Error message, if supplied configuration is incorrect.
+     * @throws IgniteException If misconfigured.
+     */
+    @Nullable public static String validateIncomingConfiguration(
```

Comment:
Since we only checking the index names then maybe we should change the method name to something like "validateIncomingIndexConfiguration(".

Reply:
ok, but i prefer to stay it as is.